### ISSUE TITLE
tests for check-ssl-hsts-preloadable.rb are failing

### ISSUE THREAD

**Issue by majormoses** [sensu-plugins/sensu-plugins-ssl#37]
These tests started failing on master and I noticed this https://github.com/sensu-plugins/sensu-plugins-ssl/blob/1.5.0/test/check-ssl-hsts-preloadable_spec.rb#L2-L5 so I am going to comment out the tests for now and ping @rwky as requested.

See: https://travis-ci.org/sensu-plugins/sensu-plugins-ssl/jobs/354638473 for example

**Comment by rwky** [comment]
Ta @majormoses I'll take a look asap.

**Comment by rwky** [comment]
I've submitted an issue upstream https://github.com/chromium/hstspreload.org/issues/147 there doesn't appear to be a domain to test with that satisfies the warnings only test.

**Comment by majormoses** [comment]
I think they remove them after a while if they have warnings such as the max age being too low. I actually had setup hsts and preloaded with them but when I checked it saw the warning and it had text about removal if not fixed so I fixed mine. I will have to look into what we can do about that. Maybe we can email the admin for https://hstspreload.org and ask them for suggestions or if it would be possible to flag an entry to keep it around with warnings. They may may have already solved this for their own testing and we can leverage that domain. Let me know if you want me to reach out or if you got it.

Here is the email I got from their website: `hstspreload AT chromium DOT org`
